_posts/blog/2018-01-03-Who's-Ang-Li.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Who's Ang Li
+categories: blog
+image:
+  feature:
+link:
+date: 2019-03-15
+modified:
+share: true
+author: Ang Li
+---
+
+## Basic information 
+
+> I’m now a PhD student at the Institute of Automation of the Chinese Academy of Sciences (CASIA) and just 
+joined Kirstie’s lab as a one-year visiting student at the Alan Turing Institute, funded by Chinese Scholarship Council (CSC). 
+**My crruent research interests include neuroscience, genetics, neuroimaging and 
+artistic data visualization, and understanding or building biomarkers for mental disorders.**
+
+
+<figure>
+  <img src="/images/Who_s_Ang_Li/jump.jpg"
+       alt="Jump">
+  <figcaption> I like challenging but sometimes I also need a man to kick my ass behind.(Photoed at Huairou District of Beijing in 2016) </figcaption>
+</figure>
+
+## Hometown 
+
+My hometown is a small city called ‘Hegang’, located at Northeast of China 
+and adjoins Russia. In winter, it is an ice wonderland, fully of beautiful snow and rime, 
+and people like having ice-suckers even when it’s minus thirty degrees outside. 
+Please don’t think that people from cold regions are cold resistant.  On the contrary, 
+people in south China can endure more cold as they don’t have heating system inside their houses. 
+As for us, in winter we sometimes open our windows to cool off as the central heating is so …! 
+(heating system is usually not intelligent but powerful in my hometown)
+
+<figure>
+  <img src="/images/Who_s_Ang_Li/heilongjiang.png"
+       alt="hometown">
+  <figcaption> My hometown (the photo from internet) </figcaption>
+</figure>
+
+## Previous experences 
+
+During my university period, I majored in automation at Harbin Institute of Technology, where I skipped most of classes because of my chair-warmer. I and my friend rent a house to set up a recording studio and I was in charge of producing MTV.  In my holiday, I did a part-time job as the figurant in Hendian,  the biggest cinema city in China, just attempted to watch how they professionally shoot videos. But after carrying on several advertising films, I realized that I shouldn’t regard video producing as my vocation, as the business often comes first than art and creativity.  
```

Comment:
```suggestion
During my university period, I majored in automation at Harbin Institute of Technology, where I skipped most of classes because I was doing so many other fun things 😬!
I and my friend rent a house to set up a recording studio and I was in charge of producing MTV.
In my holiday, I did a part-time job as the figurant in Hendian, the biggest cinema city in China, just attempted to watch how they professionally shoot videos.
But after carrying on several advertising films, I realized that I shouldn’t regard video producing as my vocation, as the business often comes first than art and creativity.  
```